SPECS/bmake/bmake.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "bmake-202010.tar.gz": "6e1261b3b194d3a92770fc73772bff052c47eada98952a0b19c4e5b7f1fe5515"
```

Comment:
Noticed this while building locally: there's a typo in the version here, it should be "20201010".

Reply:
Thanks, fixed